sbin/pfmon

```diff
@@ -391,7 +411,7 @@ run all runtasks
 sub runtasks {
     my $mask = POSIX::SigSet->new(POSIX::SIGCHLD());
     sigprocmask(SIG_BLOCK,$mask);
-    while(@REGISTERED_TASKS) {
+    while(@REGISTERED_TASKS && $process) {
```

Comment:
In that case, we should kill the child workers when we go back to slave state. 

Otherwise we'll have 2 possible slave states, one with child processes and one without.